internal/regex/regex_oniguruma.go

```diff
@@ -1,7 +1,7 @@
 package regex
 
 import (
-	"github.com/moovweb/rubex"
+	rubex "github.com/src-d/go-oniguruma"
```

Comment:
why keep the `rubex` name instead of the current go-oniguruma package name?

Reply:
...to make an integration easier? I don't know if it was a good decision. I have a feeling that either we should rename the package to `oniguruma` or rename the repo to `rubex`
WDYT?

Reply:
in other words,  the code is here: 
github.com/src-d/go-oniguruma, but the package name is still `rubex`